docs/usage.md

```diff
@@ -325,9 +344,33 @@ By default, trimmed FastQ files will not be saved to the results directory. Spec
 
 By default intermediate BAM files will not be saved. The final BAM files created after the deduplication step are always. Set to true to also copy out BAM files from the initial Bismark alignment step. If `--skip_deduplication` or `--rrbs` is specified then BAMs from the initial alignment will always be saved.
 
+### `--save_pileup_file`
+
+When running with biscuit aligner, the methylation extraction is based on vcf file. By default these vcf files will not be saved. Set to true to also copy out the vcf-file and the index-vcf file.
+
+### `--epiread`
+
+[Epiread]([https://github.com/zhou-lab/biscuit/wiki/Convert-to-epiread-format](https://github.com/zhou-lab/biscuit/wiki/Convert-to-epiread-format)) format is a compact way of storing CpG retention pattern as well as SNP information on the same read. This option will tell the biscuit workflow to generate epiread file. The SNP file used for the epiread format can be saved using `--save_snp_file`.
+
+### `--save_snp_file`
+
+SNP file created from a sample in order to be used in the epiread file is not saved in final output directory. Set to true to copy out the SNP file. 
+> **NB: The SNP detection is in progress of development within the biscuit tool**
+
+### `--soloWCGW_file`
+
+This will generatea  methylation statuses in [bedGraph](http://genome.ucsc.edu/goldenPath/help/bedgraph.html) format, intersected with soloWCGW after extracting methylation from vcf, using biscuit workflow.
```

Comment:
```suggestion
This will generate methylation statuses in [bedGraph](http://genome.ucsc.edu/goldenPath/help/bedgraph.html) format, intersected with soloWCGW after extracting methylation from vcf, using biscuit workflow.
```